src/base/test/task_test_util.h

```diff
@@ -63,10 +63,9 @@ static inline void TaskUtilPauseTest() {
     if (!d_pause_)
         return;
     std::cout << "Test PAUSED. Exit (Ctrl-d) python shell to resume";
-    std::string pythonpath;
-    if(TaskExecuteAndWait("where python", &pythonpath))
-        TaskExecuteAndWait(pythonpath, nullptr);
-
+	//std::string str = TaskExecuteAndWait("where python", true);
+    WindowsTaskExecute("python", nullptr, false);
```

Comment:
As per our Tuesday discussion, we should avoid breaking compatibility if possible.